docker-compose/generate-config.py

```diff
@@ -29,7 +29,7 @@
 ---
 services:
   mariadb:
-    image: mariadb:10.11
+    image: mariadb:10.11.10
```

Comment:
An alternate to having to having two config files is to have:

```
command: --defaults-group-suffx=.{config_path}
```

And then have a config file like:
```
[mariadb]
{common settings}

[mariadb.dev]
{dev only settings}

[mariadb.prod]
{prod only settings}
```

Reply:
Thanks for the advice, for simplicity I would keep them in separate files. Dev does not have so many **non-defaults** anyway.